app/konnect/dev-portal/access-and-approval/add-teams.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Add Developer Teams from Identity Providers
+content_type: how-to
+---
+
+In the {{site.konnect_short_name}} Dev Portal, you can import existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. This allows you to import developers automatically instead of manually copying over every single team of developers from your IdP to {{site.konnect_short_name}}.
+
+This guide explains how to map the permissions, including scopes and claims, from your group of developers in your IdP to your organization's team in {{site.konnect_short_name}}. Although this guide uses Okta, Azure Active Directory (AD), and Auth0 as examples, you can use any IdP that conforms to OIDC standards. 
+
+## Prerequisites
+
+* A test developer account in your IdP
+* An IdP configured in Konnect (OIDC or DCR) for dynamic client registration:
+    * [Okta](/konnect/dev-portal/applications/dynamic-client-registration/okta/)
+    * [Auth0](/konnect/dev-portal/applications/dynamic-client-registration/auth0/)
+    * [Azure AD](/konnect/dev-portal/applications/dynamic-client-registration/azure/)
+* [Enable app registration](/konnect/dev-portal/applications/enable-app-reg/) for Dev Portal
```

Comment:
@c3pko I think you mentioned that app reg had to be enabled, but I wasn't sure since registering an app against an API product wasn't part of this doc (users might just want developers to view APIs instead of register apps, right?)

Reply:
Good point we can skip that